js/kucoin.js

```diff
@@ -2430,10 +2435,20 @@ module.exports = class kucoin extends Exchange {
         } else {
             const request = {
                 'currency': currency['id'],
-                'from': fromId,
-                'to': toId,
                 'amount': requestedAmount,
             };
+            if (fromIsolated || toIsolated) {
+                if (this.ids.includes (fromId)) {
```

Comment:
@Dan-krm I've fixed it, but next time use something like `inArray` here otherwise we will crash in runtime using the Py version